scss/_patterns_lists.scss

```diff
@@ -13,21 +14,20 @@ $list-status-icon-height: $default-icon-size;
   %numbered-step-title {
     list-style: none;
     margin-left: 0;
+    padding-left: calc(map-get($line-heights, default-text) + $list-step-bullet-margin);
+    position: relative;
 
     &::before {
-      align-self: start;
-      background-color: $color-x-light;
-      border: 1px solid $color-x-dark;
-      border-radius: 100%;
-      content: counter(li);
+      aspect-ratio: 1 / 1;
```

Comment:
This doesn't seem to do anything

Reply:
Ah, true. I added it before I had to switch to use both width and height.

Reply:
```suggestion
```